_posts/2016-09-23-Monitoring-with-Prometheus.md

```diff
@@ -0,0 +1,667 @@
+---
+layout: post
+authors: [tom_verelst]
+title: 'Monitoring with Prometheus'
+image: /img/prometheus.jpg
+tags: [Prometheus]
+category: Monitoring
+comments: true
+---
+
+It is needless to say the world is shifting towards DevOps and microservices.
+This holy grail we all aim for adds a lot of complexity,
+as well for monitoring.
```

Comment:
ligt niet zo lekker in de mond: ofwel "especially for monitoring" of "for the general architecture as well as for monitoring"